auth.go

```diff
@@ -794,6 +869,18 @@ func (x *Central) UpdateIdentity(user *AuthUser) error {
 	return nil
 }
 
+// Unlock user in the AuthUserStore.
+func (x *Central) UnlockAccount(userId UserId) error {
+	e := x.userStore.UnlockAccount(userId)
+	if e != nil {
+		x.Log.Warnf("Failed to unlock user (%v) (%v)", userId, e)
+		return e
+	}
+
+	x.Log.Infof("Successfully unlocked user (%v)", userId)
```

Comment:
Generally, we just say "Unlocked user", ie don't say "successfully", because every log message, unless it's a failure, is implicitly success.